tools/lint/test/fixtures/test/negative_invalid_phase_type_order.js

```diff
@@ -0,0 +1,15 @@
+NEGATIVE
+^ expected errors | v input
+// Copyright (C) 2017 Mike Pennisi. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-assignment-operators-static-semantics-early-errors
+description: Minimal test
+negative:
+  type: SyntaxError
+  phase: parse
+---*/
+
+$DONOTEVALUATE();
+
+!!!
```

Comment:
thanks for adding a test. As long as we have the documentation and this, I'm good.

One nit pick: you might want to update line 3 here with 2022 and your name. For the other side, the more the test looks like the other files and the diff is only within the order, the current text is better.

Reply:
You're right, it is a copy of a similar test, that's why I didn't want to update the copyright. It's OK for me as is.